src/core/gltf2importer/meshparser_utils.cpp

```diff
@@ -75,13 +86,28 @@ QVarLengthArray<Qt3DRender::QAttribute::VertexBaseType, 3> validVertexBaseTypesF
 
 QVarLengthArray<uint, 2> validVertexSizesForAttribute(const QString &attributeName)
 {
+    {
+        // Morph Target Attributes
+        QRegularExpression re;
+        for (const QString &baseAttributeName : morphTargetBaseAttributeNames) {
+            re.setPattern(morphTargetAttributePattern.arg(baseAttributeName));
+            Q_ASSERT(re.isValid());
+            if (re.match(attributeName).hasMatch()) {
+                // Tangent attribute size is 4, all others are 3
+                if (baseAttributeName == Qt3DRender::QAttribute::defaultTangentAttributeName())
+                    return { 3, 4 };
+                return { 3 };
+            }
+        }
+    }
+
     // Standard Attributes
     if (attributeName == Qt3DRender::QAttribute::defaultPositionAttributeName())
         return { 3 };
     if (attributeName == Qt3DRender::QAttribute::defaultNormalAttributeName())
         return { 3 };
     if (attributeName == Qt3DRender::QAttribute::defaultTangentAttributeName())
-        return { 4 };
+        return { 3, 4 };
```

Comment:
why can the tangent have 3 components?

Reply:
well because the the morph target sample files are only providing 3

Reply:
Okay, so I guess is just setting the values of the tangents and the base configuration (the fourth component) is unchanged.